lib/linter.coffee

```diff
@@ -156,6 +156,34 @@ class Linter
 
     process = new BufferedProcess({command, args, options,
                                   stdout, stderr, exit})
+    process.onWillThrowError (err) =>
+      return unless err?
+      if err.error.code is 'ENOENT'
+        ignored = atom.config.get('linter.ignoredLinterErrors')
+        subtle = atom.config.get('linter.subtleLinterErrors')
+        if @linterName in subtle
+          # Show a small notification at the bottom of the screen
+          new MessagePanelView(
+            title: "#{@linterName} linter not installed"
+          ).attach()
+        else if @linterName not in ignored
+          # Prompt user, ask if they want to fully or partially ignore warnings
+          atom.confirm
+            message: "Oops, you don't have the linter #{@linterName} installed"
```

Comment:
This is my bad grammar. :grimacing: Maybe "The linter binary #{@linterName} cannot be found." I'm not sure if you want to tell them it needs to be on their `PATH` or not.